examples/live-chat/package.json

```diff
@@ -10,13 +10,16 @@
     },
     "dependencies": {
         "@faker-js/faker": "^6.0.0-alpha.5",
+        "@millicast/vue-viewer-plugin": "^0.1.0-beta.0",
```

Comment:
Keep the dependency as it was (file://), since we're going to use the last version that is not published